Error with pnpm

```
Module not found: Error: Can't resolve 'prop-types' in 'node_modules/.registry.npmjs.org/react-stripe-checkout/2.6.3/node_modules/react-stripe-checkout/dist'
```

It appears that `react` and `prop-types ` are both run-time dependencies and not build-time dependencies.

--- a/package.json
+++ b/package.json
@@ -103,6 +103,12 @@
     "url": "https://github.com/azmenak/react-stripe-checkout/issues"
   },
   "homepage": "https://github.com/azmenak/react-stripe-checkout",
+  "dependencies": {
+    "prop-types": "^15.5.8"
+  },
+  "peerDependencies": {
+    "react": ">=15"
+  },
   "devDependencies": {
     "babel-cli": "^6.11.4",
     "babel-core": "^6.11.4",
@@ -124,7 +130,6 @@
     "jsdom": "^9.4.1",
     "mocha": "^3.0.1",
     "nyc": "^7.1.0",
-    "prop-types": "^15.5.8",
     "react": "^15.5.4",
     "react-addons-test-utils": "^15.5.0",
     "react-dom": "15.5.4"

